apps/condo/domains/meter/schema/CommunalMeter.js

```diff
@@ -0,0 +1,214 @@
+const { AUTOMATIC_METER_NO_MASTER_APP, B2B_APP_NOT_CONNECTED, B2C_APP_NOT_AVAILABLE } = require('@condo/domains/meter/constants/errors')
+const { deleteReadingsOfDeletedMeter } = require('@condo/domains/meter/tasks')
+const { CommunalMeter: CommunalMeterAPI } = require('@condo/domains/meter/utils/serverSchema/index')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+
+const CommunalMeter = new GQLListSchema('CommunalMeter', {
+    schemaDoc: 'TODO DOC!',
+    fields: {
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        number: {
+            schemaDoc: 'Number of resource meter, such as "А03 9908"',
+            type: Text,
+            isRequired: true,
+            hooks: {
+                validateInput: async ({ context, operation, existingItem, resolvedData, fieldPath, addFieldValidationError }) => {
+                    // should be unique inside organization
+                    const value = resolvedData[fieldPath]
+                    let metersWithSameResourceAndNumberInOrganization
+                    if (operation === 'create') {
+                        metersWithSameResourceAndNumberInOrganization = await find('CommunalMeter', {
+                            number: value,
+                            organization: { id: resolvedData.organization },
+                            resource: { id: resolvedData.resource },
+                            deletedAt: null,
+                        })
+                    }
+                    else if (operation === 'update' && resolvedData.number !== existingItem.number) {
+                        const organization = resolvedData.organization ? resolvedData.organization : existingItem.organization
+                        const resource = resolvedData.resource ? resolvedData.resource : existingItem.resource
+
+                        metersWithSameResourceAndNumberInOrganization = await CommunalMeterAPI.getAll(context, {
+                            number: value,
+                            organization: { id: organization },
+                            resource: { id: resource },
+                            deletedAt: null,
+                        })
+                    }
+
+                    if (metersWithSameResourceAndNumberInOrganization && metersWithSameResourceAndNumberInOrganization.length > 0) {
+                        addFieldValidationError(`${UNIQUE_ALREADY_EXISTS_ERROR}${fieldPath}] Meter with same number and resource exist in current organization`)
+                    }
+                },
+            },
+        },
+
+        numberOfTariffs: {
+            type: Integer,
+            isRequired: true,
+        },
+
+        installationDate: {
+            schemaDoc: 'Date when the meter was installed in the property',
+            type: DateTimeUtc,
+        },
+
+        commissioningDate: {
```

Comment:
Please move fields, which repeat usual meter, to constants, so it can reused and maintain same logic